docs/site/components/subpage/subpage-directive.js

```diff
@@ -72,6 +72,7 @@ angular
       priority: 1500,
       restrict: 'C',
       transclude: true,
+      scope: {},
```

Comment:
Bug 2: Two `hljsClass` directive instances were sharing the same scope, so assigning the contents of the last would would overwrite all previous instances. This creates an isolate scope where they can each happily retain their contents.